### PR TITLE
Remove qwery from fill-advert-slots.js

### PR DESCRIPTION
**Pull request by ealitten**
## Does this change need to be reproduced in dotcom-rendering ?

- [x] No
- [ ] Yes (please indicate your plans for DCR Implementation)

## What is the value of this and can you measure success?

Part of https://github.com/guardian/frontend/pull/23740

### Tested

- [x] Locally
- [x] On CODE (optional)

### CODE REVIEW

**Review by marsavar — COMMENTED** [review]
Looks good! 👍 🎉

**Review by arelra — APPROVED** [review]

File: static/src/javascripts/projects/commercial/modules/dfp/fill-advert-slots.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ const fillAdvertSlots = () => {
             config.get('isDotcomRendering', false) &&
             getBreakpoint() === 'mobile'
         // Get all ad slots
-        const adverts = qwery(dfpEnv.adSlotSelector)
+        const adverts = [...document.querySelectorAll(dfpEnv.adSlotSelector)]
```
Comment (mxdvl) [inline static/src/javascripts/projects/commercial/modules/dfp/fill-advert-slots.js:38]: I think it's worth using `lib/$$` for consistency!

**Review by mxdvl — COMMENTED** [review]
That's great, do we have any plan for converting this file to TypeScript as a follow-up?

File: static/src/javascripts/projects/commercial/modules/dfp/fill-advert-slots.js (hunk 1 above)
Comment (arelra) [reply static/src/javascripts/projects/commercial/modules/dfp/fill-advert-slots.js:38]: Yep we're going to look at how to support $ and $$.

$$ does not support the use case of passing in a Bonzo element as the context which we'll need to support (or amend calling code) if we want to switch from $ to $$.

Looking at $$ it maintains an internal list of elements which might be a tad wasteful of memory when in most cases we just want to select elements from the dom.

**Review by mxdvl — APPROVED** [review]

**Pull request merged**